tools/build.sh

```diff
@@ -37,7 +53,7 @@ for DIR in src/request/*/ ; do
         cat $DIR/$url >> dist/request/$REQUEST/$JS_BUNDLE
     done
 
-    sed -i -e "s/CONFIG.ALLOWED_ORIGIN/$ALLOWED_ORIGIN/g" dist/request/$REQUEST/$JS_BUNDLE
+    replace_config_variable "CONFIG_ALLOWED_ORIGIN" "KEYGUARD_ALLOWED_ORIGIN" dist/request/$REQUEST/$JS_BUNDLE
```

Comment:
Decide on a env variable name. KEYGUARD_ALLOWED_ORIGIN is not what you document as a name in the README.